tests/workflows/test_imbalanced_join.py

```diff
@@ -0,0 +1,23 @@
+"""
+This data represents a skewed but realistic dataset that dask has been struggling with in the past.
+Workflow based on https://github.com/coiled/imbalanced-join/blob/main/test_big_join_synthetic.ipynb
+"""
+import dask.dataframe as dd
+import pytest
+
+
+@pytest.mark.client("imbalanced_join")
+def test_merge(client):
+    """Merge large df and small df"""
+    large_df = dd.read_parquet("s3://test-imbalanced-join/df1/")
+    small_df = dd.read_parquet("s3://test-imbalanced-join/df2/")
```

Comment:
Just a thought (feel free to ignore) but these datasets seem opaque to me.  I don't know how imbalanced they are, or really what this means.  Probably the answer is to go to the repository and look at notes.

An alternative would be to use `dask.datasets.timeseries` and take the random float columns and put them through some transformation to control skew / cardinality / etc..  this would put more control of the situation into this benchmark.

Again, just a thought, not a request.

Reply:
I think this would indeed be a nice utility function. Maybe even a core functionality of timeseries?
I like the idea but I suggest to decouple this from this PR